snakeAndLadderApp/src/com/workAtTech/snakeAndLadderApp/SnakeAndLadderAppLogic.java

```diff
@@ -0,0 +1,104 @@
+package com.workAtTech.snakeAndLadderApp;
+
+import com.workAtTech.snakeAndLadderApp.model.Board;
+import com.workAtTech.snakeAndLadderApp.model.Ladder;
+import com.workAtTech.snakeAndLadderApp.model.Player;
+import com.workAtTech.snakeAndLadderApp.model.Snake;
+
+import java.util.*;
+
+public class SnakeAndLadderAppLogic {
+    private static final int DEFAULT_BOARD_SIZE = 100; //Rule1 : The board will have 100 cells numbered from 1 to 100.
+    private static final int DEFAULT_NO_OF_DICES = 1;
+    private Board board;
+    private Queue<Player> players; // Players will be added one by one hence used queue
+    private int numOfDices;
+    private DiceRoll DiceRollObj;
```

Comment:
variable names should start with a small letter

Reply:
@gcnit Changed class name DiceRoll to DiceService and DiceRollObj to diceServiceObj.